packages/grafana-ui/src/components/List/AbstractList.tsx

```diff
@@ -31,11 +32,11 @@ export class AbstractList<T> extends React.PureComponent<AbstractListProps<T>> {
   }
 
   render() {
-    const { items, renderItem, getItemKey, className, inline } = this.props;
+    const { items, renderItem, getItemKey, className, inline, 'aria-labelledby': ariaLabelledBy } = this.props;
```

Comment:
🤔 I feel these changes don't belong in this PR, is it maybe a good idea to move it to another one, and keep the refactor scoped?

Reply:
Haha, yeah that's fair, got a bit carried away.